doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -15,13 +17,13 @@ The most relevant changes that are present on the master branch of the |NCS|, as
 Highlights
 **********
 
-* Added support for NFC and *pair before use* type of accessories to the Apple Find My add-on.
+There are no entries for this section yet.
```

Comment:
Please move the Find My entry to the Highlights section in release-notes-1-7-0.rst. In fact in release-notes-changelog.rst there used to be no "Highlights" section. I added it as part of the 1.7.0 release notes PR to mention "Find My".

Reply:
Please check doc/nrf/releases/release-notes-1.7.0.rst, not the changelog.